inst/pages/04_containers.qmd

```diff
@@ -755,6 +756,20 @@ Conversion is possible between other data formats. Interested readers can refer
 
 ## Exporting data container {#sec-exporting-data-container}
 
+### Transforming a TreeSummarizedExperiment into a dataframe
+
+Transforming a TreeSE object into a dataframe is straightforward with the MIA package. The `meltAssay` function is particularly handy for this purpose. It allows you to melt various parts of a TreeSE object into a dataframe based on the parameters you specify.
+
+```{r}
+molten_tse <- mia::meltAssay(tse,
+                        add_row_data = TRUE,
+                        add_col_data = TRUE,
+                        assay.type = "counts")
```

Comment:
Use the function directly without specifying the package. Also the new name of function is meltSE